advanced_source/super_resolution_with_onnxruntime.py

```diff
@@ -0,0 +1,305 @@
+"""
+Exporting a Model from PyTorch to ONNX and Running it using ONNXRuntime
+=======================================================================
+
+In this tutorial, we describe how to convert a model defined
+in PyTorch into the ONNX format and then run it with ONNXRuntime. 
+
+ONNXRuntime is a performance-focused engine for ONNX models,
+which inferences efficiently across multiple platforms and hardware
+(Windows, Linux, and Mac and on both CPUs and GPUs).
+ONNXRuntime has proved to considerably increase performance over
+multiple models as explained `here
+<https://cloudblogs.microsoft.com/opensource/2019/05/22/onnx-runtime-machine-learning-inferencing-0-4-release>`__
+
+For this tutorial, you will need to install `onnx <https://github.com/onnx/onnx>`__
+and `onnxruntime <https://github.com/microsoft/onnxruntime>`__.
+You can get binary builds of onnx and onnxrunimte with
+``pip install onnx onnxruntime``.
+Note that ONNXRuntime is compatible with Python versions 3.5 to 3.7.
+
+``NOTE``: This tutorial needs PyTorch master branch which can be installed by following
+the instructions `here <https://github.com/pytorch/pytorch#from-source>`__
+
+"""
+
+# Some standard imports
+import io
+import numpy as np
+
+from torch import nn
+import torch.utils.model_zoo as model_zoo
+import torch.onnx
+
+
+######################################################################
+# Super-resolution is a way of increasing the resolution of images, videos
+# and is widely used in image processing or video editing. For this
+# tutorial, we will first use a small super-resolution model with a dummy
+# input.
+#
+# First, let's create a SuperResolution model in PyTorch. `This
+# model <https://github.com/pytorch/examples/blob/master/super_resolution/model.py>`__
+# comes directly from PyTorch's examples without modification:
+#
+
+# Super Resolution model definition in PyTorch
```

Comment:
Add a comment about the expect input and output content (such as size, data layout, required pre/post-processing etc)